ewkcoffea/modules/selection_wwz.py

```diff
@@ -466,9 +466,9 @@ def get_mt2(w_lep0,w_lep1,met):
     nevents = len(np.zeros_like(met))
     misspart = ak.zip(
         {
-            "pt": met.pt,
+            "pt": met.pt_new,
```

Comment:
This would break if we don't have corrected MET. 

This is related to previous comment about how to better handle pt and pt_new.

Perhaps we should define a new object like MET_corrected and just pass that around (and just use .pt and .eta of that). Not sure. 

Can you make an issue to define some of these concerns so that we keep it in mind for when we do the ewkcoffea cleanups after we're approved?